docs/table_generator.py

```diff
@@ -41,4 +42,4 @@
             if f:
                 f = False
 
-    print(text)
\ No newline at end of file
+    print(text)
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline minor nitpick (and my IDE yells at me about this)

Reply:
Whoops! Should be fixed now.